### PR TITLE
SOLR-11331: Ability to Debug Solr With Eclipse IDE

### PR DESCRIPTION
**Pull request by mrkarthik**

### CODE REVIEW

**Comment by mrkarthik** [comment]
@uschindler I have managed to copy all required folder to eclipse-build, can you review the changes?

**Comment by mrkarthik** [comment]
@vthacker I have updated the PR based on your feedback

**Comment by mrkarthik** [comment]
I have updated the PR.

File: dev-tools/eclipse/dot.classpath.xsl [hunk 1]
```diff
@@ -54,7 +55,23 @@
           <classpathentry excluding="src" including="conf/**" kind="src" path="lucene/benchmark"/>
         </xsl:if>
       </xsl:for-each>
-      
+
```
Comment (uschindler) [inline dev-tools/eclipse/dot.classpath.xsl]: Could you fix the indent here. In Lucene we only allow whitespace, tabs are forbidden anywhere. This looks like it's mixed here (unfortunately I can't see it here)

Comment (uschindler) [reply dev-tools/eclipse/dot.classpath.xsl]: sure!

**Comment by mrkarthik** [comment]
Code committed by @uschindler. Commit d2152da59482b6cc338c60888b72e8f2996cb86a

**Pull request closed without merging**